src/core/rippleerror.js

```diff
@@ -42,4 +46,25 @@ util.inherits(RippleError, Error);
 
 RippleError.prototype.name = 'RippleError';
 
+
+RippleError.prototype.toString = function() {
+  let result = '[RippleError(' + this.result;
+  if (this.result_message && this.result_message !== this.result) {
+    result += ', ' + this.result_message;
+  }
+  result += ')]';
+  return result;
+};
+
+/*
+  console.log in node uses util.inspect on object, and util.inspect allows
+  to cutomize it output:
+  https://nodejs.org/api/util.html#util_custom_inspect_function_on_objects
+*/
+RippleError.prototype.inspect = function(depth) {
+  (function unused() {})(depth);
```

Comment:
This looks awkward, can you define the function in utils.js and require it from there?